pkgs/development/compilers/rust/rustc.nix

```diff
@@ -125,6 +125,10 @@ in stdenv.mkDerivation rec {
 
     # Useful debugging parameter
     # export VERBOSE=1
+  '' + optionalString stdenv.targetPlatform.isx86_32 ''
+    # From Arch: prevent SSE2 instructions in i686 output
+    sed -i "s/pentium4/pentium3/g" vendor/rustc-ap-rustc_target/src/spec/i686_unknown_linux_gnu.rs
+    sed -i "s/pentium4/pentium3/g" compiler/rustc_target/src/spec/i686_unknown_linux_gnu.rs
```

Comment:
This is unlikely to change but if it does substituteInPlace would print a warning instead of silenting failing.

Reply:
Done.